build.gradle

```diff
@@ -3,14 +3,14 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.2'
-        classpath 'me.tatarka:gradle-retrolambda:3.6.1'
+        classpath 'com.android.tools.build:gradle:3.0.0-beta2'
```

Comment:
I prefer to have a stable gradle version instead of beta. I guess this is the reason why Travis CI complains. Is it possible to have this PR without updating gradle plugin?